x-pack/plugin/inference/qa/test-service-plugin/src/main/java/org/elasticsearch/xpack/inference/mock/TestInferenceServiceExtension.java

```diff
@@ -138,6 +143,27 @@ public void infer(
             }
         }
 
+        @Override
+        public void chunkedInfer(
+            Model model,
+            List<String> input,
+            Map<String, Object> taskSettings,
+            InputType inputType,
+            ChunkingOptions chunkingOptions,
+            ActionListener<ChunkedInferenceServiceResults> listener
+        ) {
+            switch (model.getConfigurations().getTaskType()) {
+                case ANY -> listener.onResponse(makeChunkedResults(input));
+                case SPARSE_EMBEDDING -> listener.onResponse(makeChunkedResults(input));
```

Comment:
nit: My ide was telling me before that I think `ANY` and `SPARE_EMBEDDING` can be combined into one case.